data-sources/cloudflare/config.yml

```diff
@@ -0,0 +1,35 @@
+id: cloudflare
+displayName: Cloudflare Network Logs
+description: |
+  ## About Cloudflare
+
+  Cloudflare is an industry leading global network designed to make everything you connect to the Internet secure, private, fast, and reliable. Protect and accelerate external, public-facing web properties; secure your internal operations on a single global network; and build new applications on our serverless platform. Internet properties powered by Cloudflare have all web traffic routed through its intelligent global network, which gets smarter with every request.
+
+  ## About this quickstart
+
+  Cloudflare is on a mission to help build a better Internet. The Cloudflare quickstart will enable you to monitor and analyze web traffic metrics on a dashboard, integrating with New Relic’s database to provide an at-a-glance overview of the most important logs and metrics from your websites and applications.
+
+  For more information check out our website at [cloudflare.com](https://www.cloudflare.com/)
+icon: logo.png
+install:
+  primary:
+    nerdlet:
+      nerdletId: nr1-install-newrelic.quickstart-installation-plan
+      nerdletState:
+        quickstart_id: fc2bb0ac-6622-43c6-8c1f-6a4c26ab5434
```

Comment:
Can we double check this is the right nerdlet state value? I thought this was camelCase, but I could be wrong.

Reply:
@jerelmiller You are correct! It looks like camelCase is the way to go, I'll get these updated.

Reply:
All data sources should be updated to use camelCase